docs/org-owners.html

```diff
@@ -8,14 +8,14 @@
 </h3>
 
 <div class="row">
-	<div class="col-main">
-		<div class="chart-container">
-		    <table class="table" data-url="{{ site.dataURL }}/org-owners.tsv"></table>
-		</div>
-	</div>
-	<div class="col-aside">
-		<div class="info-box">
-			<p>Owners for each GitHub organization.</p>
-		</div>
-	</div>
+    <div class="col-main">
+        <div class="chart-container">
+            <table class="table" data-url="{{ site.dataURL }}/org-owners.tsv"></table>
```

Comment:
Sorry, I should have been more specific. We’re using tabs in the other files, and it would be best to keep that consistent.